server/controllers/votes/VoteController.js

```diff
@@ -0,0 +1,116 @@
+import models from '../../models';
+import {
+  ARTICLE_LIKED_MSG,
+  ARTICLE_DISLIKED_MSG,
+  ARTICLE_UNLIKED_MSG
+} from '../../helpers/responseMessages';
+
+const { ArticleVote } = models;
+/**
+ * @description The vote controller class
+ * @class
+ */
+class Vote {
+  /**
+   * @description Hanldes Like functionality
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} Response
+   */
+  static async likeAnArticle(req, res, next) {
+    try {
+      const { articleId } = req.params;
+      const userId = req.user.id;
+
+      const dbResult = await ArticleVote.findOrCreate(
+        { where: { userId: userId, articleId: articleId } }
+      );
+      if(!dbResult[1]) {
+        Vote.updateArticleVote(1, userId, articleId);
+        return res.status(202).json({
+          message: ARTICLE_LIKED_MSG
+        });
+      }
+      return res.status(201).json({
+        message: ARTICLE_LIKED_MSG
+      });
+    } catch (error) {
+      return next(error);
+    }
+  }
+
+  /**
+   * @description Handles the dislike functionality
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} Response
+   */
+  static async dislikeAnArticle(req, res, next) {
+    try {
+      const userId = req.user.id;
+      const { articleId } = req.params;
+
+      const dbResult = await ArticleVote.findOrCreate(
+        { where: { userId: userId, articleId: articleId } }
```

Comment:
what are we passing as the vote object?

Reply:
Yeah, we should pass `0` back when the vote is not defined. Added this to the table models now. Will push ASAP.